src/test/java/io/getunleash/streaming/StreamingFeatureFetchingTest.java

```diff
@@ -1,4 +1,4 @@
-package io.getunleash.integration;
+package io.getunleash.streaming;
```

Comment:
Not sure why this was in the integration package. It's alone in there, doesn't match it's actual package structure and that doesn't follow the pattern for our test packages anyway